src/header/internal.tsx

```diff
@@ -55,14 +55,7 @@ export default function InternalHeader({
       ref={__internalRootRef}
     >
       <div className={clsx(styles.main, styles[`main-variant-${variantOverride}`], isRefresh && styles.refresh)}>
-        <div
-          className={clsx(
-            styles.title,
-            styles[`title-variant-${variantOverride}`],
-            isRefresh && styles.refresh,
-            description && [styles[`root-has-description`]]
```

Comment:
If I understand this correctly, when `description` was present, `.root-has-description` was applied, which removed the `padding-bottom`. Why don't we need this anymore?

Reply:
The class was applied twice, once on the title (this instance) and the other on the root. The styles only referenced the class applied on the root, so this one was never actually used.